packages/ember-glimmer/tests/integration/syntax/let-test.js

```diff
@@ -0,0 +1,367 @@
+import { get, set } from 'ember-metal';
+import { A as emberA, ObjectProxy, removeAt } from 'ember-runtime';
+import { moduleFor, RenderingTest } from '../../utils/test-case';
+import { strip } from '../../utils/abstract-test-case';
+import { EMBER_TEMPLATE_BLOCK_LET_HELPER } from 'ember/features';
+
+if (EMBER_TEMPLATE_BLOCK_LET_HELPER) {
+  moduleFor('Syntax test: {{#let as}}', class extends RenderingTest  {
+
+    templateFor({ cond, truthy, falsy }) {
+      return `{{#let ${cond} as |test|}}${truthy}{{else}}${falsy}{{/let}}`;
+    }
+
+    ['@test keying off of `undefined` does not render']() {
```

Comment:
I find the description a little hard to parse. Perhaps `@test a key with an undefined value renders nothing`?

Reply:
Oops, the tests were adapted from `{{with}}` and this should not actually pass 🤔 .

Reply:
It passes because `undefined` renders as ``, hah. Adapted the test for `{{let}}`.